www/faq.php

```diff
@@ -75,38 +75,52 @@
 
 	<tr><td class="textcontent">
 	<p>
-	You should be familiar with the <a href="https://www.freebsd.org/doc/en_US.ISO8859-1/books/handbook/ports.html">Ports</a>
-	section of <a href="https://www.freebsd.org/doc/en_US.ISO8859-1/books/handbook/index.html">The FreeBSD Handbook</a>.
-	Pay careful attention to the difference between a port and a package.
+	Pay careful attention to the differences between a port and a package.
+	<p>
+	Blogged by the FreeBSD Foundation in August 2024:
+	<a href="https://freebsdfoundation.org/blog/freebsd-ports-and-packages-what-you-need-to-know/">
+	FreeBSD Ports and Packages: What you need to know</a>. More generally,
+	please be familiar with the <a href="https://docs.freebsd.org/en/books/handbook/ports/">
+	Ports</a> section of the <a href="https://docs.freebsd.org/en/books/handbook/">
+	FreeBSD Handbook</a>.
 
 	<h2>Definitions for the hopelessly lazy</h2>
 
 	<h3 id="port">What is a port?</h3>
 	<p>
-	If you install the port, the source will be downloaded, patched if necessary,
```

Comment:
This section contains a highly simplified overview of ports and packages. We have glossed over many details. More experienced users may feel compelled to point out misleading or incorrect points. Instead, they should submit patches whilst keeping the first sentence in mind.

Reply:
I'm happy with a fair amount of glossing-over, because: 

- **Definitions for the hopelessly lazy** is a light-hearted heading
- we plead with the reader to be familiar with the ports-related chapter of the FreeBSD Handbook
- the Foundation's blog post concludes with a link to the same chapter
- the [last part of the chapter](https://docs.freebsd.org/en/books/handbook/ports/#ports-broken) has "Fix it!" as point 3 of 4.